core/src/broadcast_stage.rs

```diff
@@ -387,10 +387,15 @@ pub fn broadcast_shreds(
     let mut shred_select = Measure::start("shred_select");
     let packets: Vec<_> = shreds
         .iter()
-        .map(|shred| {
+        .filter_map(|shred| {
             let broadcast_index = weighted_best(&peers_and_stakes, shred.seed());
+            let node = &peers[broadcast_index];
 
-            (&shred.payload, &peers[broadcast_index].tvu)
+            if is_global(&node.tvu) {
+                Some((&shred.payload, &node.tvu))
+            } else {
+                None
+            }
```

Comment:
@behzadnouri mind reviewing the conflict resolution of this block?

Reply:
lgtm, thank you for doing the backport.